docker-compose.yml

```diff
@@ -41,11 +41,9 @@ services:
         build:
             context: ./backend
             dockerfile: Dockerfile
-            args:
-                - DATABASE_URL=${ELYSIA_DATABASE_URL}
-                - RESEND_API_KEY=${ELYSIA_RESEND_API_KEY}
-        environment:
-            - NODE_ENV=production
         restart: always
         ports:
-            - 3000:3000
\ No newline at end of file
+            - 3000:3000
+        env_file:
+            - .env
+            
```

Comment:
Don't change production server docker-compose file without a reason, you gonna break everything

Reply:
fixed